src/components/Dropdown/Dropdown.jsx

```diff
@@ -64,7 +64,8 @@ const Dropdown = ({
   ref,
   withMandatoryDefaultOptions,
   isOptionSelected,
-  insideScroll
+  insideOverflowContainer,
```

Comment:
i know i changed the name but i guess it's ok because it was released only from Wednesday... I guess no one is start to used it already